Assignments/Object oriented/Final_Project_customer.js

```diff
@@ -0,0 +1,95 @@
+CB.CloudApp.init('yegguroxoyua', '47345f24-5127-4318-abd3-b054fe11a2a2');
+
+class customer{
+	constructor(username,password){
+		this.user = username;
+		this.pass = password;
+		console.log(this.pass);
+		console.log(this.user);
+	}
+
+	signup() {
+		var user = this.user;
+		var pass = this.pass;
+		var error = "no";
+		var query = new CB.CloudQuery("customer");
+		console.log(query);
+		query.find({
+			success: function(list){
+				console.log(list);
+				for(let x=0; x<list.length;x++){
```

Comment:
Another way we can write this is using `for` `of`. 

```js
for (const item of list) {
  console.log(list.document.username);
}
``` 

It'll iterate through the array and return the item inside.